### PR TITLE
Notify on channel access disconnections.

### PR DESCRIPTION
**Pull request by willrogers**
Previously no update was made for channel access disconnections.

### CODE REVIEW

File: coniql/caplugin.py [hunk 1]
```diff
@@ -127,6 +128,13 @@ def channel_from_update(
                 )
                 self.cached_status = status
 
+        if connected is not None:
```
Comment (thomascobb) [inline coniql/caplugin.py:131]: Why not just use time_value.ok instead of passing connected?

Comment (willrogers) [reply coniql/caplugin.py:131]: Mostly just a relic from an earlier version. You could say it's more explicit though: it means we are tightly coupled to the monitor that was created outside of the class. True already though.

**Comment by willrogers** [comment]
Meanwhile (a hangover from the last PR) we could simplify: since we are waiting to construct our `CAChannelMaker` and we never update `writeable` we could set it in the constructor.

File: coniql/caplugin.py (hunk 1 above)
Comment (willrogers) [reply coniql/caplugin.py]: OK, I have changed this in the latest commit.

File: coniql/caplugin.py [hunk 2]
```diff
@@ -160,10 +157,8 @@ async def get_channel(
             cainfo(pv, timeout=timeout),
         )
         # Put in channel id so converters can see it
```
Comment (willrogers) [inline coniql/caplugin.py:158]: What does this comment mean?

File: tests/test_caplugin.py [hunk 3]
```diff
@@ -159,6 +159,46 @@ async def test_get_enum_pv(engine: Engine, ioc: Popen):
...
+        }
+        status {
+            quality
+        }
+    }
+}
+"""
+        % PV_PREFIX
+    )
+    results = []
```
Comment (willrogers) [inline tests/test_caplugin.py:179]: Why does Mypy require a type hint here? Why is it even checking this function? It seems entirely equivalent to `test_subscribe_ticking()`.

**Pull request merged**